modules/nf-core/rtgtools/pedfilter/tests/nextflow.config

```diff
@@ -1,7 +1,4 @@
 process {
-
-    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
-
     withName: "test_rtgtools_pedfilter_vcf_output:RTGTOOLS_PEDFILTER" {
```

Comment:
This won't configure anything since that process does not exist anymore :) You'll need to manually add this config to the vcf tests

Reply:
Thank you @nvnieuwk 

I have tried to fix this.